iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -194,6 +197,10 @@ public HiveIcebergMetaHook(Configuration conf) {
 
   @Override
   public void preCreateTable(org.apache.hadoop.hive.metastore.api.Table hmsTable) {
```

Comment:
is that an empty impl? that won't be a backward compatible change

Reply:
Fixed.